WorkloadManagementSystem/DB/SandboxMetadataDB.py

```diff
@@ -348,10 +348,10 @@ def getSandboxId(self, SEName, SEPFN, requesterName, requesterGroup, field='SBId
       return S_ERROR("No sandbox matches the requirements")
     return S_OK(data[0][0])
 
-  def getSandboxOwner(self, SEName, SEPFN, requesterName, requesterGroup):
+  def getSandboxOwner(self, SEName, SEPFN, requesterDN, requesterGroup):
     """ get the sandbox owner, if such sandbox exists
```

Comment:
Since you are are it, can you add the parameters in the docstring? Also for `getSandboxId`.

Reply:
OK